scripts/ci/test_automation.sh

```diff
@@ -54,4 +54,7 @@ az -v
 az -h
 
 title 'Running tests'
+
+pip install git+https://github.com/haroldrandom/azure-python-devtools.git@fix-python-3.8.3
```

Comment:
Will revert it once https://github.com/Azure/azure-python-devtools/pull/65 is merged